src/core_plugins/kibana/public/dashboard/listing/dashboard_landing_page_table.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { LandingPageTable } from 'ui_framework/components';
+import { getDashboardColumns } from './get_dashboard_columns';
+import { ItemTableActions } from 'ui/saved_object_table/item_table_actions';
+import { DashboardConstants } from '../dashboard_constants';
+
+export function DashboardLandingPageTable({ itemTableState, onDeleteItems, onFilterItems, isFetchingItems }) {
```

Comment:
Take my suggestions with a grain of salt... :) I think the yellow flag here is a symptom of a problem, but not a root cause of problems.

> If this encapsulated the <ControlledTable>...</ControlledTable> (which looks like a good idea to me) it would probably appear less "superfluous".

I agree.